docs/documentation/admin/admin_bots.md

```diff
@@ -1,58 +1,158 @@
-
-The Routine builder is explained in the [Routine Section](/docs/documentation/automation/admin_routine).
+
+<div className="container box">
+
+<div className="row table-row-title"><b>General Information</b></div>
+
+<div className="row table-row-1">
+<div className="col col--3"><b>Name:</b></div>
+<div className="col col--5">The bot's display name</div>
+<div className="col col--4"><em>It doesn't have to be unique.</em></div>
+</div>
+
+<div className="row table-row-2">
+<div className="col col--3"><b>User bot name:</b></div>
+<div className="col col--5">
+
+The _user bot name_ is the name of the _user_ and email address that is automatically created to represent and relate the bot within the Cotalker environment.
+
+</div>
+<div className="col col--4"><em>API requests for the bot must use this name. The bot's access roles are assigned to the bot's created 'user'.</em></div>
+</div>
+
+<div className="row table-row-1">
+<div className="col col--3"><b>Description:</b></div>
+<div className="col col--5">Description of what the bot does.</div>
+<div className="col col--4"><em>Displayed on the 'bots settings panel'.</em></div>
+</div>
+
+<div className="row table-row-title">
+<div className="col col--12"><b>Access</b></div>
+</div>
+
+<div className="row table-row-2">
+<div className="col col--3"><b>Access Roles:</b></div>
+<div className="col col--5">
+
+The access roles define what the bot's _user_ can do in the routine.
+
+</div>
+<div className="col col--4"><em>Attention: A 'bot' can run a routine without proper access roles but will not render the desired results.</em></div>
+</div>
+
+<div className="row table-row-title">
+<div className="col col--12"><b>Functions</b></div>
+</div>
+
+<div className="row table-row-1">
+<div className="col col--3"><b>Global:</b></div>
+<div className="col col--5">
+
+Allows a _bot_ to work across all groups and channels in the company.
+
+</div>
+<div className="col col--4"><em></em></div>
+</div>
+
+<div className="row table-row-2">
+<div className="col col--3"><b>It's a survey command:</b></div>
+<div className="col col--5">
+
+This switch indicates that a _survey_ initializes the _bot_.
+
+</div>
+<div className="col col--4"><em></em></div>
+</div>
+
+<div className="row table-row-1">
+<div className="col col--3"><b>Form:</b></div>
+<div className="col col--5">
+
+Indicates the _survey_ that initializes the _bot_. Any _survey_ can be selected.
+
+</div>
+<div className="col col--4"><em>This field only appears when the 'It's a survey command' option is activated.</em></div>
+</div>
+
+<div className="row table-row-2">
+<div className="col col--3"><b>Save form response:</b></div>
+<div className="col col--5">
+
+Saves the information sent through the _survey_ as a [COTMessage](/docs/documentation/api/communication/messages) in its [context](/docs/documentation/automation/triggers_and_contexts#channel-survey-trigger). You can access the data with COTLang script, e.g., `$VALUE#messages`.
```

Comment:
@Mauriceac 

If `saves the information` is set, then the messages (COTMessage Array) are **added** to the context response as `messages`, and can be read with COTLang with `$VALUE#messages`. 

NOTE: This feature is deprecated, now messages retrieved by `form.id` can be obtained by comparing this with  `answer.formId`

Reply:
## Review: @acedward 
- Edited according to latest indications.
- **Question:** Where can I send users to find more information about *form.id* and  *answer.formId*.
![Screen Shot 2021-07-05 at 2 56 24 PM](https://user-images.githubusercontent.com/47286214/124510255-6b069a00-dda1-11eb-9122-97dc5d879161.png)

Reply:
@Mauriceac This isn't explained anywhere now.

1. We should update the User Model with user.form description
`message.form.id` contains the main-survey id, unique for the entire survey
`message.form.subid` contains the sub-survey id, unique per-sub-survey - and null for the non-sub-survey parts.
 
2. We should add where we introduce Surveys, that each one is built by messages, and these fields are automatically set.

Reply:
3. We should update Answer model with `answer.formid` This is a copy of the messages `message.form.id` value

Reply:
> 1. We should update the Message Model with message.form description...

![Screen Shot 2021-07-07 at 5 43 09 PM](https://user-images.githubusercontent.com/47286214/124832892-5534d880-df4b-11eb-81e8-7128e9d37258.png)

> 2. We should add where we introduce Surveys, that each one is built by messages, and these fields are automatically set.

![Screen Shot 2021-07-07 at 5 43 48 PM](https://user-images.githubusercontent.com/47286214/124832729-1a32a500-df4b-11eb-98f0-72376da767e6.png)

> 3. We should update Answer model with answer.formId This is a copy of the messages message.form.id value

![Screen Shot 2021-07-07 at 5 44 25 PM](https://user-images.githubusercontent.com/47286214/124832945-65e54e80-df4b-11eb-8b1d-29bd1e25faaf.png)